python/ray/_private/internal_api.py

```diff
@@ -232,3 +235,12 @@ def f():
             return
 
         worker.core_worker.free_objects(object_refs, local_only)
+
+
+def get_local_lineage_reconstruction_tasks() -> List[
+    Tuple[common_pb2.LineageReconstructionTask, int]
+]:
+    """Returns Map[LineageReconstructionTask, count] as a list of pair."""
```

Comment:
Update this with something more descriptive? like "Return the ongoing retry tasks triggered by lineage reconstruction"